### DISCUSSION TITLE
Cart page custom js not loading correctly

### DISCUSSION THREAD

**Discussion by JamesDenish** [breezefront/community#196] [1. Help]
I  have added custom.js on cart page through which I am adding classes and other required changes it loads and add the class wherever needed but unfortunately it stops working and classes get disappeared from that place which where added if mouse events occur.

Kindly reply @vovayatsyuk I am trying but not getting any solution. It's urgent.

Thanks

**Comment by vovayatsyuk** [comment]
custom.js (from breeze docs) will not load on breeze-disabled pages. Did you disable breeze on the cart page?

**Comment by JamesDenish** [comment]
@vovayatsyuk 
Yes but I have added it in magento 2 manner but still facing issue.
Please help
Thanks
